Modules/getpath.c

```diff
@@ -322,7 +322,7 @@ search_for_prefix(const _PyCoreConfig *core_config,
     /* Check to see if argv[0] is in the build directory */
     wcsncpy(prefix, calculate->argv0_path, MAXPATHLEN);
     prefix[MAXPATHLEN] = L'\0';
-    joinpath(prefix, L"Modules/Setup");
+    joinpath(prefix, L"Modules/config.c");
```

Comment:
Hmm, perhaps this is not okay for the PC build?  Since we don't copy Setup.dist to Setup, the check would fail without a change.  We could look for Setup.local but looking for config.c makes more sense to me.

Reply:
[Talking to myself, ;-)]
It looks like this should be okay.  The PC build has its own getpath.c file and so this change doesn't affect it.  The purpose of this check (as I understand) is to find if we are running 'python'  from within the build directory.  So, we use a file that exists in the build directory (and if separate source and build directories are used, not in the source directory).